www/templates/default/html/js/events.js

```diff
@@ -4,6 +4,15 @@ require(['jquery', 'wdn', 'modernizr'], function($, WDN, Modernizr) {
 	var $progress = $('<progress>'),
 		mqBp2 = '(min-width: 768px)';
 	
+	$(document).ready(function() {
```

Comment:
ah, yes. This was my first dive into JS for this so yeah, manager js sounds good. I'll get on it.